lib/web_ui/dev/felt.dart

```diff
@@ -22,6 +23,7 @@ CommandRunner<bool> runner = CommandRunner<bool>(
   ..addCommand(BuildCommand())
   ..addCommand(CleanCommand())
   ..addCommand(CreateSimulatorCommand())
+  ..addCommand(GenerateFallbackFontDataCommand())
```

Comment:
This is more of a roll process, similar to `canvaskit_roller.dart` or `browser_roller.dart`, than something part of our normal dev cycle. So I'm not sure if a `felt` command is needed, but I don't feel strongly about it.

More importantly, let's document it in our README.md. The best place would we next to our existing "Rolling X" sections.

Reply:
Done